apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -177,6 +179,42 @@ export const updateDocuments = async ({ body }, response) => {
 	response.send(results);
 };
 
+/**
+ * @type {import('express').RequestHandler<{id: number}, any, any, any>}
```

Comment:
is the payload an array of giud strings? should it be declared in the RequestHandler type here?